src/utility.ts

```diff
@@ -43,26 +44,27 @@ export async function showErrorMessage(message: IMessage, ...items: string[]): P
 
 export async function showInformationMessageWithTroubleshooting(message: IMessage, ...items: string[]): Promise<string | undefined> {
     const choice = await showInformationMessage(message, ...items, LEARN_MORE);
-    return handleTroubleshooting(choice, message.message);
+    return handleTroubleshooting(choice, message.message, message.anchor);
 }
 
 export async function showWarningMessageWithTroubleshooting(message: IMessage, ...items: string[]): Promise<string | undefined> {
     const choice = await showWarningMessage(message, ...items, LEARN_MORE);
-    return handleTroubleshooting(choice, message.message);
+    return handleTroubleshooting(choice, message.message, message.anchor);
 }
 
 export async function showErrorMessageWithTroubleshooting(message: IMessage, ...items: string[]): Promise<string | undefined> {
     const choice = await showErrorMessage(message, ...items, LEARN_MORE);
-    return handleTroubleshooting(choice, message.message);
+    return handleTroubleshooting(choice, message.message, message.anchor);

```

Comment:
rename IMessage/message  to  ITroubleShooting/troubleShoot?

Reply:
fixed.